app/permissions/decidim/department_admin/permissions.rb

```diff
@@ -119,7 +119,7 @@ def has_permission?(requested_action)
 
           # NEWSLETTER
           -> { permission_for?(requested_action, :admin, :index, :newsletter) },
-          -> { permission_for?(requested_action, :admin, :read, :newsletter, restricted_rsrc: context[:newsletter]) },
+          -> { permission_for?(requested_action, :admin, :read, :newsletter) },
```

Comment:
shouldn't we check if newsletter's author is from the same area as current's?

Reply:
No. As we talked face to face, everyone could see the newsletter, but only update or send if they has the same area than current_user